packages/flutter/test/foundation/leak_tracking.dart

```diff
@@ -19,27 +69,18 @@ typedef LeaksObtainer = void Function(Leaks foundLeaks);
 /// 1. Listens to [MemoryAllocations] events.
 /// 2. Uses `tester.runAsync` for leak detection if [tester] is provided.
 ///
-/// If you use [testWidgets], pass [tester] to avoid async issues in leak processing.
-/// Pass null otherwise.
-///
-/// Pass [leaksObtainer] if you want to get leak information before
-/// the method failure.
-Future<void> withFlutterLeakTracking(
-  DartAsyncCallback callback, {
-  required WidgetTester? tester,
-  StackTraceCollectionConfig stackTraceCollectionConfig =
-      const StackTraceCollectionConfig(),
-  Duration? timeoutForFinalGarbageCollection,
-  LeaksObtainer? leaksObtainer,
-}) async {
-  // The method is copied (with improvements) from
-  // `package:leak_tracker/test/test_infra/flutter_helpers.dart`.
-
-  // The method is not combined with [testWidgets], because the combining will
-  // impact VSCode's ability to recognize tests.
-
+/// Pass [config] to troubleshoot leaks. See [LeakTrackingTestConfig] for details.
+Future<void> _withFlutterLeakTracking(
+  DartAsyncCallback callback,
+  WidgetTester tester,
+  LeakTrackingTestConfig config,
+) async {
   // Leak tracker does not work for web platform.
   if (kIsWeb) {
+    if (!_webWarningPrinted) {
+      _webWarningPrinted = true;
+      debugPrint('Leak tracking is not supported on web platform.');
+    }
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#only-log-actionable-messages-to-the-console

Reply:
Made actionable. Peer PR in leak tracker: https://github.com/dart-lang/leak_tracker/pull/48. Please, review.